scripts/cibuild

```diff
@@ -23,6 +23,9 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
     if [ "${1:-}" = "--help" ]; then
         usage
     else
+        # Ensure container images are current
```

Comment:
Since the Python dependencies listed in `requirements.txt` are baked into the container, and the container image isn't rebuilt before running tests on CI, tests were failing because the `rollbar` module was missing.

Reply:
When I tested this locally, the image produced by this command without `-f docker-compose.ci.yml` is tagged with `:latest`. It seems like we want the image built by this command to be tagged with the Git commit SHA, so that we can avoid scenarios were `:latest` is being built by one CI job, but tested by another.

It is possible that we need to tweak the body of `test_django` in `test` to test with the appropriate image in CI (but also the appropriate image when run locally).

Reply:
What if we just build the tagged `django` container image twice?